format.md

```diff
@@ -124,6 +133,29 @@ is a list of xy vertices. The child name of the outer list should be "polygons";
 the child name of the middle list should be "rings"; the child name of the
 inner list should be "vertices".
 
+### Serialized encodings
+
+**Well-known binary (WKB)**: `Binary` or `LargeBinary` or `FixedSizeBinary`
+
+It may be useful for implementations that already have facilities to read
+and/or write well-known binary (WKB) to store features in this form without
+modification. When well-known binary is stored in an Arrow array, it should
+follow the convensions defined in the
+[GeoParquet specification](https://github.com/opengeospatial/geoparquet).
+Notably, it should use the ISO form instead of EWKB when Z or M dimensions
+are included and axis order is defined as easting, northing/longitude, latitude
```

Comment:
```suggestion
are included and axis order is defined as easting, northing or longitude, latitude
```

That reminds me, we should probably add a similar section to this page as the "Coordinate axis order" from https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#coordinate-axis-order. Because also for the native encodings this matters (well, at least for the interleaved), and also here it can contrast with the CRS (defined in the extension metadata)